code/modules/surgery/organs/organ_internal.dm

```diff
@@ -266,36 +266,6 @@
 				H.adjustFireLoss(-cursed_heart.heal_burn)
 				H.adjustOxyLoss(-cursed_heart.heal_oxy)
 
-/obj/item/organ/internal/lungs
-	name = "lungs"
-	icon_state = "lungs"
-	gender = PLURAL
-	organ_tag = "lungs"
-	parent_organ = "chest"
-	slot = "lungs"
-	vital = 1
-
-//Insert something neat here.
-///obj/item/organ/internal/lungs/remove(mob/living/carbon/M, special = 0)
-//	owner.losebreath += 10
-	//insert oxy damage extream here.
-//	. = ..()
-
-
-/obj/item/organ/internal/lungs/on_life()
-	..()
-	if(germ_level > INFECTION_LEVEL_ONE)
```

Comment:
it is stealthy to everyone else around them, doctors and any one with knowledge has no way to know a patient has a lung infection, yes it makes it stealthy